source-oracle/discovery.go

```diff
@@ -268,18 +267,18 @@ SELECT t.owner, t.table_name, t.column_id, t.column_name, t.nullable, t.data_typ
     ON (t.owner = c.owner AND t.table_name = c.table_name AND t.column_name = c.column_name)`
 
 type oracleColumnType struct {
-	original  string
-	length    int
-	scale     int16
-	precision int16
-	t         reflect.Type
-	format    string
-	jsonType  string
-	nullable  bool
+	Original  string
+	Length    int
+	Scale     int16
+	Precision int16
+	T         reflect.Type
+	Format    string
+	JsonType  string
+	Nullable  bool
```

Comment:
Made these public so debug logging will be able to log these values, they are very useful for debugging